spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/TaskDefinitionController.java

```diff
@@ -354,4 +355,10 @@ public TaskExecutionAwareTaskDefinition apply(TaskDefinition source) {
 			}
 		}
 	};
+
+	private void validateTaskName(String name) {
```

Comment:
Can we add some comments as to why 52?

Reply:
> Can we add some comments as to why 52?

For this the exact count isn't finalized, I'll add a comment for now but once I raise this PR, the devs will discuss to figure out exactly what to have as max length.

Reply:
Perfect